presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -761,6 +778,35 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
             tableProperties.put(AVRO_SCHEMA_URL_KEY, validateAndNormalizeAvroSchemaUrl(avroSchemaUrl, hdfsContext));
         }
 
+        // Textfile specific properties
+        Integer textHeaderSkipCount = getTextHeaderSkipCount(tableMetadata.getProperties());
```

Comment:
nit: use `Optional<Integer>`

Reply:
There is also `OptionalInt`. I'm not sure which is better here, just a reminder :)

Reply:
`Optional<Integer>` is better, `OptionalInt` is missing `map` method which is useful.

Reply:
We had this discussion already. Since the classes cannot be used easily together (eg missing .map method), we decided to use generic classes everywhere except where there is compelling performance advantage from using specialized classes